scss/variables/modal.js

```diff
@@ -0,0 +1,75 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis", "./colors", "./typograpy" ], factory );
+	} else if ( typeof exports === "object" ) {
+		require( "./chassis" );
+		require( "./colors" );
+		module.exports = factory( require( "./typography" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.modal = {
+
+	"shadows": {
+		name: "Modal shadow styles",
+		value: {
+			"base": "0px 8px 11px 0px rgba(0,0,0,0.15)"
+		}
+	},
+	"borders": {
+		name: "Modal border styles",
+		value: {
+			"default": "1px solid " + chassis.colors.grey.value.base,
+			"radius": "3px"
+		}
+	},
+	"padding": {
+		name: "Modal Padding value",
+		value: {
+			"padding": "15px"
+		}
+	},
+	"element": {
+		name: "Generic modal styles",
+		value: {
+			"padding": "0px",
+			"margin": "10px",
+			"background": chassis.colors.background.value.base
+		}
+	},
+	"title": {
+		name: "Modal Title Styles",
+		value: {
+			"font-size": "24px",
+			"font-weight": "700",
+			"margin": "0"
```

Comment:
CSS style guide for jQuery states to not use units for 0s. https://contribute.jquery.org/style-guide/css/



> On Jun 7, 2016, at 6:35 PM, Kristy Yeaton <notifications@github.com> wrote:
> 
> In scss/variables/modal.js:
> 
> > +		}
> > +	},
> > +	"element": {
> > +		name: "Generic modal styles",
> > +		value: {
> > +			"padding": "0px",
> > +			"margin": "10px",
> > +			"background": chassis.colors.background.value.base
> > +		}
> > +	},
> > +	"title": {
> > +		name: "Modal Title Styles",
> > +		value: {
> > +			"font-size": "24px",
> > +			"font-weight": "700",
> > +			"margin": "0"
> We should be consistent about how we write 0: 0 vs 0px (the rest of this file seems to be using "0px") I prefer using 0 myself but we should talk about it in our next meeting maybe?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>